### DISCUSSION TITLE
Afther first install and reboot no connect to Birdnetip.local

### DISCUSSION THREAD

**Discussion by NvBgm** [mcguirepr89/BirdNET-Pi#1252] [Q&A] [answered]
After installing as the manual indicates and the automatic reboot of the PI, I cannot reach BirnetIP from the browser. My hostname is in BirdnetIP, so I should start the site with https:\\birdnetip.local, but unfortunately it doesn't work.
Connecting to Putty and WinSCP is no problem.
Anyone have an idea??

**Comment by ferherlo** [comment]
Hi, 2 days with the same problem, with Raspberry Pi OS Lite (64-bit) Debian Bookworm no desktop environment. The default homepage I’m seeing is the Apache2. I am following the instalation guide https://github.com/mcguirepr89/BirdNET-Pi/wiki/Installation-Guide

Help, please!

**Comment by blogdemoi** [comment]
Hi,

I advice you to use this fork https://github.com/Nachtzuster/BirdNET-Pi which has Bookworm support. This fork carry on with the wonderful work that @mcguirepr89 did with BirdnetPi.

**Comment by ferherlo** [comment, marked as answer]
Apologies for my limited knowledge. Could you guide me on how to install (https://github.com/Nachtzuster/BirdNET-Pi ) that on my Raspberry Pi? Thank you

**Reply by blogdemoi** [reply]
> Apologies for my limited knowledge. Could you guide me on how to install (https://github.com/Nachtzuster/BirdNET-Pi ) that on my Raspberry Pi? Thank you

I did an upgrade from my mcguirepr89 to the fork install.

However, for a fresh install you should just follow the same procedure: https://github.com/mcguirepr89/BirdNET-Pi/wiki/Installation-Guide

Until 19:

> Copy and paste the command below into the terminal and press "Enter"
> Note: Ctrl+Shift+V will often paste in Terminals -- in PuTTY, right-click to paste

where you should input the Nachtzuster repo:

`curl -s https://raw.githubusercontent.com/Nachtzuster/BirdNET-Pi/main/newinstaller.sh | bash`

Then carry on from 20) with: https://github.com/mcguirepr89/BirdNET-Pi/wiki/Installation-Guide

and you will be using Nachtzuster fork.

**Reply by ferherlo** [reply]
Perfect, works! Thank you!
